specification/containerservices/resource-manager/Microsoft.ContainerService/stable/2018-03-31/managedClusters.json

```diff
@@ -0,0 +1,1290 @@
+          "description": "Storage profile specifies what kind of storage used. Choose from StorageAccount and ManagedDisks. Leave it empty, we will choose for you based on the orchestrator choice."
+        },
+        "vnetSubnetID": {
+          "$ref": "#/definitions/ContainerServiceVnetSubnetID",
+          "description": "VNet SubnetID specifies the vnet's subnet identifier. If you specify either master VNet Subnet, or agent VNet Subnet, you need to specify both. And they have to be in the same VNet."
+        },
+        "maxPods": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Maximum number of pods that can run on a node.",
+          "default": 30
+        },
+        "osType": {
+          "$ref": "#/definitions/OSType",
+          "description": "OsType to be used to specify os type. Choose from Linux and Windows. Default to Linux."
+        }
+      },
+      "required": [
+        "name",
+        "vmSize"
+      ],
+      "description": "Profile for the container service agent pool."
+    },
+    "ContainerServiceWindowsProfile": {
+      "properties": {
+        "adminUsername": {
+          "type": "string",
+          "description": "The administrator username to use for Windows VMs.",
+          "pattern": "^[a-zA-Z0-9]+([._]?[a-zA-Z0-9]+)*$"
+        },
+        "adminPassword": {
+          "type": "string",
+          "description": "The administrator password to use for Windows VMs.",
+          "pattern": "^(?=.*[a-z])(?=.*[A-Z])(?=.*[!@#$%\\^&\\*\\(\\)])[a-zA-Z\\d!@#$%\\^&\\*\\(\\)]{12,123}$"
+        }
+      },
+      "required": [
+        "adminUsername",
+        "adminPassword"
+      ],
+      "description": "Profile for Windows VMs in the container service cluster."
+    },
+    "ContainerServiceLinuxProfile": {
+      "properties": {
+        "adminUsername": {
+          "type": "string",
+          "description": "The administrator username to use for Linux VMs.",
+          "pattern": "^[a-z][a-z0-9_-]*$"
+        },
+        "ssh": {
+          "$ref": "#/definitions/ContainerServiceSshConfiguration",
+          "description": "SSH configuration for Linux-based VMs running on Azure."
+        }
+      },
+      "required": [
+        "adminUsername",
+        "ssh"
+      ],
+      "description": "Profile for Linux VMs in the container service cluster."
+    },
+    "ContainerServiceNetworkProfile": {
+      "properties": {
+        "networkPlugin": {
+          "type": "string",
+          "enum": [
+            "azure",
+            "kubenet"
+          ],
+          "default": "kubenet",
+          "x-ms-enum": {
+            "name": "NetworkPlugin",
+            "modelAsString": true
+          },
+          "description": "Network plugin used for building Kubernetes network."
+        },
+        "networkPolicy": {
+          "type": "string",
+          "enum": [
+            "calico",
+            "cilium"
```

Comment:
Can you remove cilium from the enum here, it is my mistake originally and I was told cilium does not need to be supported in the coming release.